templates/restoreList.html

```diff
@@ -27,20 +27,51 @@ <h2>Restore custom backup list</h2>
   }
 </script>
 
-
 <form name="restoreListForm" onsubmit="return OnSubmitForm();">
-  Target dataset id (optional):<br> <input type="text" name="targetDatasetId" value=""><br>
-  Backup list<span style="color: red;">*</span>:<br><textarea name="requestBody" placeholder='
-[
+  <div class="form-group row">
+    <label for="targetDatasetId" class="col-sm-2 col-form-label">Target dataset id</label>
+    <div class="input-group col-sm-8">
+      <div class="input-group-prepend">
+        <span class="input-group-text" id="projectAddOn">{{ restoration_project_id }}.</span>
+      </div>
+      <input type="text" class="form-control" aria-describedby="projectAddOn" id="targetDatasetId"
+             name="targetDatasetId" placeholder="source_project_id___source_dataset_id">
+      <small id="targetDatasetIdHelp" class="form-text text-muted">Optional. Temporary dataset id
+        that will be used (and created if does not exist) as container for restored table. Remember
+        that this will be a temporary dataset with expiration time set to 7 days. Note that passed
+        dataset could already exists - it should be in the same location as backup. If target
+        dataset id is not passed, then source dataset id value will be used as a target dataset id
+        in restoration project
```

Comment:
"In case of restoring backups from different datasets multiple target datasets will be created. " 

it is important for list restoration

Reply:
Fixed